externs/olx.js

```diff
@@ -2399,6 +2399,7 @@ olx.interaction;
 /**
  * Interactions for the map. Default is `true` for all options.
  * @typedef {{altShiftDragRotate: (boolean|undefined),
```

Comment:
Doesn't it need a new line? like [#6681](https://github.com/openlayers/openlayers/pull/6681/commits/639f40cb94adaae3ef18bc9dbecfba38ff404d40)